pyfar/signals/stochastic.py

```diff
@@ -26,23 +16,25 @@ def noise(n_samples, spectrum="white", rms=1, sampling_rate=44100, seed=None):
     spectrum : str, optional
         'white' to generate noise with constant energy across frequency.
         'pink' to generate noise with constant energy across filters with
-        constant relative bandwith.
+        constant relative bandwith. The default is 'white'.
     rms : double, array like, optional
         The route mean square (RMS) value of the noise signal. A multi channel
         noise signal is generated if an array of RMS values is passed.
         The default is 1.
     sampling_rate : int, optional
         The sampling rate in Hz. The default is 44100.
-    seed : int, None
+    seed : int, None, optional
         The seed for the random generator. Pass a seed to obtain identical
         results for multiple calls of white noise. The default is None, which
         will yield different results with every call.
 
     Returns
     -------
-    signal : pyfar.Signal
+    signal : Signal
         The noise as a Signal object. The Signal is in the time domain and
-        has the 'rms' FFT normalization (see pyfar.fft.normalization).
+        has the 'rms' FFT normalization (see
+        :py:func:`pyfar.dsp.fft.normalization`). The spectrum and rms amplitude
```

Comment:
Maybe `type` is a more precise expression than `spectrum`?

Reply:
added information to make it more clear :)